dockertest/environment_unittests.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+# Pylint runs from a different directory, it's fine to import this way
+# pylint: disable=W0403
+
+import unittest
+
+class FakeCheckOutput(object):
+
+    exception = None
+    callcount = 0
+    command = None
+    name = None
+    version = '1.0'
+    release = 'a'
+    arch = '8088'
+    output_fmt = '%s#%s#%s#%s\n'
+    allnvras = tuple()
```

Comment:
I think this should probably be `None`, but I do kind of like the idea of initializing it with a dummy value as a hint to the maintenance programmer. In that case, though, it should be `[tuple()]` (list), shouldn't it?